Src/PRuntimes/PSymbolicRuntime/src/main/java/psymbolic/commandline/PSymOptions.java

```diff
@@ -26,6 +31,74 @@ public class PSymOptions {
                 .build();
         options.addOption(inputFile);
 
+        // project name
+        Option projectName = Option.builder("p")
+                .longOpt("project")
+                .desc("Name of the project")
+                .numberOfArgs(1)
+                .hasArg()
+                .argName("Project Name (string)")
+                .build();
+        options.addOption(projectName);
+
+        // read program state from file
+        Option readFromFile = Option.builder("r")
+                .longOpt("read")
+                .desc("Name of the file with the program state")
+                .numberOfArgs(1)
+                .hasArg()
+                .argName("File Name (string)")
+                .build();
+        options.addOption(readFromFile);
+
+        // Enable writing the program state to file
+        Option writeToFile = Option.builder("w")
+                .longOpt("write")
+                .desc("Enable writing program state")
+                .numberOfArgs(0)
+                .build();
+        options.addOption(writeToFile);
```

Comment:
What is the default location for writing the file, can you specify the file to read from as well or it searching in a default location?

Reply:
When `--write` command-line argument is passed, output files (there can be multiple of them, one each for each backtracking point) are written in the relative `output/` directory, same as where search statistics and trace files are written.

When reading from a file, the file is provided from the command-line as `--read <path-to-file>`, e.g., `--read ../somepath/filename.out`

Reply:
Thanks!